manifests/base/service.yaml

```diff
@@ -8,12 +8,16 @@ metadata:
     prometheus.io/port: "8080"
   labels:
     app: training-operator
-  name: training-operator
+  name: training-operator-metrics-service
```

Comment:
So that Service is actually removed by the RHOAI overlay, see https://github.com/opendatahub-io/training-operator/commit/005c2a02d59daf0f9028f149c8d2270deb0a1b14.

A PodMonitor is created, which should meet SRE monitoring needs. What is the actual issue that you're trying to solve?

Reply:
This is the issue: [RHOAIENG-6431](https://issues.redhat.com/browse/RHOAIENG-6431)
I was following a comment by wen 


> See requirement:
> 
> https://issues.redhat.com/browse/RHOAIENG-6819?focusedId=24691555&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-24691555 (this is general for all component)